packages/legacy-workbench/src/pages/repositories.html

```diff
@@ -67,17 +67,21 @@ <h1>
                                 <tbody>
                                 <tr class="repository"
                                     ng-repeat="repository in getRepositoriesFromLocation(location.uri) | orderBy: ['type === \'system\'', 'location', 'id'] track by repository.hash"
-                                    ng-class="{'table-info text-secondary': isRepoActive(repository)}">
-                                    <td width="25">
+                                    ng-class="{'table-info text-secondary': isRepoActive(repository)}"
+                                    guide-selector="repository-{{repository.id}}">
+                                    <td class="select-repository-button" guide-selector="select-repository-button-wrapper">
                                         <span role="button" class="icon-connection-off icon-lg text-secondary m-0"
                                               gdb-tooltip="{{'connect.repo' | translate}}" tooltip-placement="top"
                                               ng-click="setRepository(repository)"
                                               ng-if="!isRepoActive(repository)"
                                               data-repository-id="{{repository}}"
-                                              guide-selector="repository-{{repository.id}}-plug"></span>
+                                              guide-selector="select-repository-button">
```

Comment:
The parent element has a class `select-repository-button` and here you have again attribute with the same value. This is confusing.

Reply:
I can rename the parent class to match the guide selector `select-repository-button-wrapper.` What do you think?